dev-libs/rocm-device-libs/rocm-device-libs-6.0.0.ebuild

```diff
@@ -25,15 +23,17 @@ HOMEPAGE="https://github.com/RadeonOpenCompute/ROCm-Device-Libs"
 LICENSE="MIT"
 SLOT="0/$(ver_cut 1-2)"
 IUSE="test"
-RESTRICT="!test? ( test )"
+RESTRICT="strip !test? ( test )"
```

Comment:
You merged shadowed `RESTRICT="strip"`, but why it was needed? `rocm-device-libs` is a bunch of `*.bc` files, there is just nothing to strip anyways?